src/views/Home.vue

```diff
@@ -6,7 +6,7 @@
     </div>
     <div class="home__content">
       <Title class="home__header" msg="Which Anime Character Are You?" />
-      <Button class="home__button" text="Start Quiz" />
+      <Button class="home__button" text="Start Quiz" @click="firstPage" />
```

Comment:
It would be better to make `<Button />` behave as an `<a />` if there is a `to` prop:

```
<Button to="/quiz/1" />
```

which would mean making your button into a conditional component:

```
<Component :is="to ? 'a' : 'button' />
```

A small suggestion that maybe could be revisited. For now though this solution works nicely.

Reply:
I tested this and found that the `is` attribute on HTML element has been removed in Vue.js v3+

For more info see: [https://eslint.vuejs.org/rules/no-deprecated-html-element-is.html](url)